spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
+  before :each do
+    @orphan_statuses = OrphanStatus.all
+    @orphan_sponsorship_statuses = OrphanSponsorshipStatus.all
+    @provinces = Province.all
+    @orphan = build_stubbed :orphan
+  end
+
+  specify 'has a form' do
+    render
+
+    assert_select 'form'
+  end
+
+  describe '"Cancel" button' do
+    specify 'when :id' do
+      allow(@orphan).to receive(:id).and_return 42
+      render
+
+      assert_select 'a[href=?]', hq_orphan_path(42), text: 'Cancel'
+    end
+  end
+
+  specify 'form values' do
+    render
+
+    #fextfields
+    ["name", "father_given_name", "family_name", "mother_name", "date_of_birth",
+     "minor_siblings_count", "sponsored_minor_siblings_count"].each do |field|
+      assert_select "input#orphan_#{field}" do
+        assert_select "[value=?]", CGI::escape_html(@orphan[field].to_s)
+      end
```

Comment:
We discussed in the last nights meeting that these escaping issues will be fixed in a different task. (all other new and edit views have the same problem).
I noticed there was already a task for this: https://osraav.atlassian.net/browse/OSRA-372